### PR TITLE
[MAD-761] Split nylas

### PR DESCRIPTION
**Pull request by gothlike**

### CODE REVIEW

File: build-files/config.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
+  "GOOGLE_OAUTH_CLIENT_ID": "",
+  "GOOGLE_OAUTH_CLIENT_SECRET": "",
+
+  "BLOCK_ENCRYPTION_KEY": "",
+  "SECRET_ENCRYPTION_KEY": "",
+
+  "NYLAS_ADMIN_PASSWORD": "bbb4a88d392370c66a51512c4403353a386653f341bebdeaa030d289d8889cdc:e4e3670c6e0f4174b2a6acc2a33acf69",
```
Comment (gothlike) [inline build-files/config.json:8]: this password is for local development only. It's overwritten for production environment

File: build-files/config.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    "ENABLED": false,
+    "URL": "",
+    "APP_ROLE_ID": "",
+    "APP_ROLE_SECRET_ID": ""
+  },
+
+  "DATABASE_USERS": {
+    "mysql": {
+      "USER": "root",
+      "PASSWORD": "deepframetest"
```
Comment (gothlike) [inline build-files/config.json:25]: Will be changed soon when Nylas starts using MySQL in AWS.

File: inbox/api/auth_api.py [hunk 3]
```diff
@@ -105,11 +105,12 @@ def new_gmail_account():
     auth_info['provider'] = 'gmail'
 
     email_address = auth_info['email']
-    account_exsist = False
+    account_exists = False
```
Comment (gothlike) [inline inbox/api/auth_api.py:108]: fixed typo.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+# Doing pip upgrade setuptools leaves behind this problematic symlink
+RUN rm -rf /usr/lib/python2.7/dist-packages/setuptools.egg-info
+
+RUN pip uninstall -y chardet
+
+RUN pip install -r requirements.txt
+
+COPY ./build-files/config.json /etc/inboxapp/config.json
+COPY ./build-files/secrets.json /etc/inboxapp/secrets.json
```
Comment (gothlike) [inline Dockerfile:95]: This file is included into **.gitignore**.
It should be created manually before first build&run.

File: build-files/config.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
+{
+  "GOOGLE_OAUTH_CLIENT_ID": "",
```
Comment (gothlike) [inline build-files/config.json:2]: must be overridden in **secrets.json**. I will add a template file **secrets.template.json** with all required properties that should be filled in for **local** development.

File: build-files/config.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  "GOOGLE_OAUTH_CLIENT_SECRET": "",
```
Comment (gothlike) [inline build-files/config.json:3]: must be overridden in **secrets.json**. I will add a template file **secrets.template.json** with all required properties that should be filled in for **local** development.

File: build-files/config.json (hunk 5 above)
Comment (AlexanderBabchenko) [reply build-files/config.json:2]: I think it's nice to leave the same comment in the code.

File: build-files/config.json (hunk 6 above)
Comment (AlexanderBabchenko) [reply build-files/config.json:3]: I think it's nice to leave the same comment in the code.

File: build-files/config.json (hunk 1 above)
Comment (AlexanderBabchenko) [reply build-files/config.json:8]: I think it's nice to leave the same comment in the code.

**Review by AlexanderBabchenko — APPROVED** [review]
It would be nice if you also added exactly what was changed in commit in future. (http://take.ms/Cg2hM)

https://chris.beams.io/posts/git-commit/

**Review by iderets — APPROVED** [review]

**Pull request merged**